### PR TITLE
update Python minimum version to 3.8

### LINKED ISSUE #3368: Update docs to reflect Python 3.7 is no longer supported
### Contributions

- [X] I have read the contribution docs, and understand what's expected of me.

### Link to the page on docs.getdbt.com requiring updates

- https://github.com/dbt-labs/docs.getdbt.com/blob/ede186c4759847a2eae4c157d32b53354c3ac2b3/website/docs/faqs/Core/install-pip-os-prereqs.md?plain=1#L26
- https://github.com/dbt-labs/docs.getdbt.com/blob/ede186c4759847a2eae4c157d32b53354c3ac2b3/website/docs/faqs/Core/install-python-compatibility.md?plain=1#L3

### What part(s) of the page would you like to see updated?

Minimum Python version should be bumped to 3.8 as described in https://github.com/dbt-labs/dbt-core/issues/7082

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by sdebruyn**
## What are you changing in this pull request and why?
Docs change for PR https://github.com/dbt-labs/dbt-core/pull/7623 / issue https://github.com/dbt-labs/dbt-core/issues/7082

resolves #3368

## Checklist
- [x] Add versioning components, as described in [Versioning Docs](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#versioning-entire-pages)
- [ ] Add a note to the prerelease version [Migration Guide](https://github.com/dbt-labs/docs.getdbt.com/tree/current/website/docs/guides/migration/versions)
- [x] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) and [About versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) so my content adheres to these guidelines.
- [x] Add a checklist item for anything that needs to happen before this PR is merged, such as "needs technical review" or "change base branch."

### CODE REVIEW

File: website/docs/faqs/Core/install-pip-os-prereqs.md [hunk 1]
```diff
@@ -33,7 +39,7 @@ python --version
 
 ```
 
-If you need a compatible version, you can download and install [Python version 3.7 or higher for MacOS](https://www.python.org/downloads/macos).
+If you need a compatible version, you can download and install [Python version 8 or higher for MacOS](https://www.python.org/downloads/macos).
```
Comment (dbeatty10) [inline website/docs/faqs/Core/install-pip-os-prereqs.md]:
```suggestion
If you need a compatible version, you can download and install [Python version 3.8 or higher for MacOS](https://www.python.org/downloads/macos).
```

File: website/docs/faqs/Core/install-python-compatibility.md [hunk 2]
```diff
@@ -1,13 +1,19 @@
 ---
 title: What version of Python can I use?
-description: "Python versions 3.7 and newer can be used with dbt Core"
+description: "Python versions 8 and newer can be used with dbt Core"
```
Comment (dbeatty10) [inline website/docs/faqs/Core/install-python-compatibility.md]:
```suggestion
description: "Python versions 3.8 and newer can be used with dbt Core"
```

**Review by dbeatty10 — CHANGES_REQUESTED** [review]
Thanks for raising this PR @sdebruyn !

This review has a couple suggestions that I will commit in a moment.

**Review by dbeatty10 — DISMISSED** [review]
We haven't made a drop-down menu item for the 1.6 version of the docs yet, but I confirmed in the preview that it didn't create problems to include this ahead of time so it is ready when the time comes.

https://deploy-preview-3369--docs-getdbt-com.netlify.app/faqs/core/install-python-compatibility?version=1.6

Could you be extra confirmation @mirnawong1 ?

**Comment by sdebruyn** [comment]
> Thanks for raising this PR @sdebruyn !
> 
> This review has a couple suggestions that I will commit in a moment.

Sorry, looks like I wasn't wearing my glasses while typing those `3.8`s 😂

File: website/docs/faqs/Core/install-pip-os-prereqs.md [hunk 3]
```diff
@@ -55,6 +61,12 @@ pip install cryptography~=3.4
 
 Windows requires Python and git to successfully install and run dbt Core.
 
+<VersionBlock firstVersion="1.6">
+Install [Git for Windows](https://git-scm.com/downloads) and [Python version 3.8 or higher for Windows](https://www.python.org/downloads/windows/).
```
Comment (mirnawong1) [inline website/docs/faqs/Core/install-pip-os-prereqs.md:66]:
```suggestion

Install [Git for Windows](https://git-scm.com/downloads) and [Python version 3.8 or higher for Windows](https://www.python.org/downloads/windows/).
```

File: website/docs/faqs/Core/install-pip-os-prereqs.md [hunk 4]
```diff
@@ -55,6 +61,13 @@ pip install cryptography~=3.4
 
 Windows requires Python and git to successfully install and run dbt Core.
 
+<VersionBlock firstVersion="1.6">
+
+Install [Git for Windows](https://git-scm.com/downloads) and [Python version 3.8 or higher for Windows](https://www.python.org/downloads/windows/).
+</VersionBlock>
+
+<VersionBlock lastVersion="1.5">
 Install [Git for Windows](https://git-scm.com/downloads) and [Python version 3.7 or higher for Windows](https://www.python.org/downloads/windows/).
```
Comment (mirnawong1) [inline website/docs/faqs/Core/install-pip-os-prereqs.md:72]:
```suggestion

Install [Git for Windows](https://git-scm.com/downloads) and [Python version 3.7 or higher for Windows](https://www.python.org/downloads/windows/).
```

Comment (mirnawong1) [inline website/docs/faqs/Core/install-pip-os-prereqs.md:72]:
```suggestion

Install [Git for Windows](https://git-scm.com/downloads) and [Python version 3.7 or higher for Windows](https://www.python.org/downloads/windows/).
```

**Comment by mirnawong1** [comment]
This is awesome, thank you for opening this up @sdebruyn and for looking it over @dbeatty10 ! I made some small tweaks because the linked text wasn't rendering correctly due to spacing (see screenshot) but [it looks](https://deploy-preview-3369--docs-getdbt-com.netlify.app/faqs/core/install-pip-os-prereqs.md) great now! confirm the drop down works fine and doesn't harm the user experience. approved on my end, wdyt @dbeatty10 ?

Screenshot 2023-05-16 at 12 16 43

**Review by mirnawong1 — APPROVED** [review]

**Review by mirnawong1 — APPROVED** [review]

**Review by dbeatty10 — APPROVED** [review]

**Comment by dbeatty10** [comment]
Looks great to meet @mirnawong1 -- squashing and merging.

**Pull request merged**